skipper.go

```diff
@@ -1292,6 +1295,38 @@ func getRedisUpdaterFunc(namespace, name string, kdc *kubernetes.Client) func()
 	}
 }
 
+func updateEndpointsFromURL(address string) func() []string {
+	return func() []string {
+		/* #nosec */
+		resp, err := http.Get(address)
+		if err != nil {
+			log.Errorf("failed to connect to redis endpoint %v, due to: %v", address, err)
```

Comment:
in all error cases you can return nil to exit early or you likely end up in panic